app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -84,7 +83,7 @@ module?.exports = React.createClass
       return true if not requiredSelects.length
 
       select = (i) ->
-        return i if annotation.value[i]?.value isnt null
+        return i if annotation.value[i]?.value isnt null and annotation.value[i]?.value isnt undefined
```

Comment:
CoffeeScripts' `?` operator checked `null` and `undefined` for you, so this could be `annotation.value[i]?.value?`.

But we can actually simplify this whole function quite a bit:

```coffee
isAnnotationComplete: (task, annotation) ->
  task.selects.every (select, i) ->
    not select.required or annotation.value[i]?.value?
```